src/meta-srv/src/election/postgres.rs

```diff
@@ -147,10 +213,13 @@ impl PgElection {
         client: Client,
         store_key_prefix: String,
         candidate_lease_ttl_secs: u64,
+        table_name: String,
+        lock_id: u64,
     ) -> Result<ElectionRef> {
+        let sql_factory = ElectionSqlFactory::new(lock_id, table_name.clone());
```

Comment:
We can move `table` into the new fn
```suggestion
        let sql_factory = ElectionSqlFactory::new(lock_id, table_name);
```